tests/test_helper.tcl

```diff
@@ -100,6 +100,7 @@ set ::all_tests {
     unit/cluster/hostnames
     unit/cluster/multi-slot-operations
     unit/cluster/slot-ownership
+    unit/cluster/links
```

Comment:
@madolson @hpatro this one was missing from test_helper. please ack that it should be added.

Reply:
Yes, that looks like a miss.